rpc/src/rpc.rs

```diff
@@ -250,6 +248,7 @@ impl JsonRpcRequestProcessor {
             CommitmentLevel::SingleGossip | CommitmentLevel::Confirmed => unreachable!(), // SingleGossip variant is deprecated
         };
 
+        let r_bank_forks = self.bank_forks.read().unwrap();
```

Comment:
Maybe add a comment at line 250 saying any operations above might delay/block the block commit so only include relevant code?